app/src/models/diff.ts

```diff
@@ -118,18 +120,27 @@ export interface IBinaryDiff {
   readonly kind: DiffType.Binary
 }
 
-export interface IDiffTooLarge {
-  readonly kind: DiffType.TooLarge
-  /**
-   * The length of the diff output from Git which exceeds the runtime limits:
-   *
-   * 268435441 bytes = 256MB - 15 bytes
-   */
-  readonly length: number
+export interface ILargeTextDiff {
+  readonly kind: DiffType.LargeText
+  /** The unified text diff - including headers and context */
+  readonly text?: string
+  /** The diff contents organized by hunk - how the git CLI outputs to the caller */
+  readonly hunks?: ReadonlyArray<DiffHunk>
```

Comment:
does the `hunks` value need to be optional? looks like the only time we call it, it is set